File: 3069-distribute-elements-into-two-arrays-i/3069-distribute-elements-into-two-arrays-i.py
class Solution:
    def resultArray(self, nums: List[int]) -> List[int]:
        arr1=[]
        arr1.append(nums[0])

        arr2=[]
        arr2.append(nums[1])
        for i in range(2,len(nums)):
            if arr1[len(arr1)-1]>arr2[len(arr2)-1]:
                arr1.append(nums[i])
            else:
                arr2.append(nums[i])
        return arr1+arr2

File: 3069-distribute-elements-into-two-arrays-i/3069-distribute-elements-into-two-arrays-i.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var resultArray = function(nums) {
    let arr1 = [];
    let arr2 = [];
    arr1.push(nums[0]);
    arr2.push(nums[1]);
        for(let i=2; i<nums.length; i++){
            if(arr1[arr1.length-1] > arr2[arr2.length-1]){
                arr1.push(nums[i]);
            }else{
                arr2.push(nums[i]);
            }
        }    
    return arr1.concat(arr2);
};